sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Samples/BindingToSessionMessageActions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests.Samples
+{
+    public static class BindingToSessionMessageActions
+    {
+        #region Snippet:ServiceBusBindingToSessionMessageActions
+        [FunctionName("BindingToSessionMessageActions")]
+        public static async Task Run(
+            [ServiceBusTrigger("<queue_name>", Connection = "<connection_name>", IsSessionsEnabled = true)]
+            ServiceBusReceivedMessage[] messages,
+            ServiceBusSessionMessageActions sessionActions)
+        {
+            foreach (ServiceBusReceivedMessage message in messages)
+            {
+                if (message.MessageId == "1")
+                {
+                    await sessionActions.DeadLetterMessageAsync(message);
+                }
+                else
+                {
+                    await sessionActions.CompleteMessageAsync(message);
+                }
+            }
+
+            await sessionActions.SetSessionStateAsync(new BinaryData("<session state>"));
```

Comment:
It may be helpful to drop a comment here about what this does and why I'd be interested in it.